lib/models/discount.freezed.dart

```diff
@@ -275,37 +276,31 @@ class _$_Discount implements _Discount {
   @override
   bool operator ==(dynamic other) {
     return identical(this, other) ||
-        (other is _Discount &&
-            (identical(other.identifier, identifier) ||
-                const DeepCollectionEquality()
-                    .equals(other.identifier, identifier)) &&
-            (identical(other.price, price) ||
-                const DeepCollectionEquality().equals(other.price, price)) &&
-            (identical(other.priceString, priceString) ||
-                const DeepCollectionEquality()
-                    .equals(other.priceString, priceString)) &&
-            (identical(other.cycles, cycles) ||
-                const DeepCollectionEquality().equals(other.cycles, cycles)) &&
-            (identical(other.period, period) ||
-                const DeepCollectionEquality().equals(other.period, period)) &&
-            (identical(other.periodUnit, periodUnit) ||
-                const DeepCollectionEquality()
-                    .equals(other.periodUnit, periodUnit)) &&
-            (identical(other.periodNumberOfUnits, periodNumberOfUnits) ||
-                const DeepCollectionEquality()
-                    .equals(other.periodNumberOfUnits, periodNumberOfUnits)));
+        (other.runtimeType == runtimeType &&
+            other is _Discount &&
+            const DeepCollectionEquality()
```

Comment:
was the idential() call no longer necessary?

Reply:
This code is autogenerated and we forgot to generate it when increasing the version of freezed in 3.9.1

Reply:
I guess not? the `.freezed` files are all auto-generated by freezed

Reply:
👍